Colours.txt

```diff
@@ -158,19 +139,6 @@
 #define smrWhite 13494510
 #define smrYellow 65535
 
-;Stand & Hold Labels
-#define smrBBlabel 13814651
-#define standHold 16777215
-
 ;Background Colours
-#define blackBackground 0
-#define blueBackground 3932160
-
-;Mil SMR
-#define smrMilGrass 4551749
-#define smrMilBoundary 6579300
-#define smrMilRoad 6908265
-#define smrMilApron 11908533
-#define smrMilRunway 4934475
-#define smrMilTaxiway 9474192
-#define smrMilBuilding 9142113
+#define Black 0
+#define Blue 3932160
```

Comment:
Thinking about it now, it seems a bit odd we have these background colours separate at then end, maybe have them in the middle? If not just part of the main colours are the top. If not the latter, perhaps we should think of a name for the top section? Although things like "apron" and "building" etc. seem like SMR colours? So it doesn't seem as simple as Radar colours and SMR colours. Wonder what your thoughts are on this.

Also, should these two be capitalised? Seems to go against the convention used elsewhere of only capitalising first letters if abreviations or ICAO code parts.

Reply:
> Thinking about it now, it seems a bit odd we have these background colours separate at then end, maybe have them in the middle? If not just part of the main colours are the top. If not the latter, perhaps we should think of a name for the top section? 

Yeah. I left them as that because it was there before. They are both used as a background and in radar/smrs

Reply:
> Also, should these two be capitalised? Seems to go against the convention used elsewhere of only capitalising first letters if abreviations or ICAO code parts

Can change this. I think a few others like "Edge" will need to be changed manually.